data/templates/article-wrapper.mst

```diff
@@ -15,6 +15,7 @@
         <script type="text/javascript" src="resource:///com/endlessm/knowledge/data/templates/js/{{{.}}}"></script>
         {{/javascript-files}}
 
+        <script type="application/json" id="content-metadata">{{{content-metadata}}}</script>
```

Comment:
You could consider making this a `<script type="text/javascript">` and prefixing the JSON object with something like `window.contentMetadata = `. That would make it much easier to access it from browser-side JS, but a disadvantage is that it would pollute the global namespace.